modules/ROOT/pages/faq.adoc

```diff
@@ -81,7 +81,11 @@ https://lists.fedoraproject.org/archives/list/coreos@lists.fedoraproject.org/[co
 * Twitter at https://twitter.com/fedora[@fedora] (all Fedora and
 other relevant news)
 
-https://discussion.fedoraproject.org/t/launch-faq-what-are-the-communication-channels-around-fedora-coreos/46/1[Discuss on discussion.fedoraproject.org]
+There is a community meeting that happens every Wednesday at 16:30 UTC.  See the https://apps.fedoraproject.org/calendar/CoreOS[Fedora CoreOS fedocal] for the most up-to-date information.
```

Comment:
I wonder if we should just say "happens every week" and let the link to the fedocal do the heavy lifting. This would prevent rot if we ever changed it and forgot to update it here.